src/cli.py

```diff
@@ -39,18 +42,32 @@ def cli():
                 # Can be deleted for flight
                 print("calling __setPIPE")
                 gs.__setPIPE__()
+                data = bytearray(libcsp.packet_get_data(toSend))
+
+            if (
+                server == sysVals.APP_DICT.get('OBC') and 
+                port == sysVals.SERVICES.get('SCHEDULER').get('port') and
+                (data[0] == sysVals.SERVICES.get('SCHEDULER').get('subservice').get('SET_SCHEDULE').get('subPort') or
+                data[0] == sysVals.SERVICES.get('SCHEDULER').get('subservice').get('DELETE_SCHEDULE').get('subPort') or
+                data[0] == sysVals.SERVICES.get('SCHEDULER').get('subservice').get('REPLACE_SCHEDULE').get('subPort'))
```

Comment:
How would 'REPLACE_SCHEDULE' be handled differently compared to 'SET_SCHEDULE' (Does the scheduler file need to look different?) , and how would 'DELETE_SCHEDULE' be handled? All these three seem to follow the same process of reading a schedule file so I'm just wondering if I need to do something special

Reply:
@yunqingyi

Reply:
REPLACE_SCHEDULE deletes the existing one, then starts a new schedule. SET_SCHEDULE adds onto the existing schedule